public/stylesheets/cards.css

```diff
@@ -128,3 +132,82 @@ fieldset {
 #continue-card-title {
   font-size: 25px;
 }
+
+@media (max-width: 1024px) {
+  .card-title {
+    font-size: 25px;
+  }
+  #gameGuesses > div {
+    margin-bottom: 8px;
+  }
+  .setting-fieldset > label {
+    margin-right: 15px;
+  }
+  #gameGuesses {
+    font-size: 13px;
+  }
+  #page-counter {
+    font-size: 20px;
+  }
+  .page-title {
+    font-size: 30px;
+  }
+  .card-button {
+    font-size: 19px;
+  }
+  .prev-session-info {
+    font-size: 16px;
+  }
+}
+
+@media (max-width: 576px) {
+  .page-title {
+    font-size: 19px;
+    padding: 5px;
+  }
+  .setting {
+    padding: 6px 15px;
+  }
+  .setting-fieldset {
+    font-size: 11px;
+  }
+  .setting-title {
+    font-size: 14px;
+  }
+  #page-counter {
+    font-size: 13px;
+    padding: 0px 10px;
+  }
+  .card-title {
+    font-size: 18px;
+  }
+  .card-button {
+    font-size: 15px;
+  }
+  #gameGuesses {
+    font-size: 12px;
+  }
+  .field-input-div {
+    padding: 40px 0px;
```

Comment:
This does nothing

Reply:
Actually, it does. It prevents you from selecting very easy difficulty on lower resolutions, for example - 440px width.